easyInterface/Diffraction/Interface.py

```diff
@@ -60,7 +60,7 @@ def default(cls) -> 'LoggedUndoableDict':
     @classmethod
     def fromPars(cls, experiments: Union[Experiments, Experiment, List[Experiment]],
                  phases: Union[Phases, Phase, List[Phase]],
-                 calculations: Optional[Union[Calculations, Calculation, List[Calculation]]] = {}) -> 'LoggedUndoableDict':
+                 calculations: Optional[Union[Calculations, Calculation, List[Calculation]]] = {}):
```

Comment:
Same as above, why was the hint removed?

Reply:
See above